Userland/Services/SystemServer/ServiceManagementServer.ipc

```diff
@@ -4,4 +4,7 @@ endpoint ServiceManagementServer = 4151813
 
     ServiceList() => (Vector<String> services)
     ServiceStatus(String service_name) => (String status)
+    ServiceSetEnabled(String service_name, bool enabled) => ()
+    ServiceStart(String service_name) => ()
+    ServiceStop(String service_name) => ()
```

Comment:
It makes sense to either make both accept booleans (`SetEnabled`, `SetRunning`), or make 4 distinct methods.